app/Http/routes.php

```diff
@@ -27,6 +27,8 @@
     Route::resource('competitions', 'CompetitionsController');
 
     // Waiting rooms routes.
+    Route::get('waitingrooms/{waitingrooms}/split', 'WaitingRoomsController@showSplitForm')->name('split');
+    Route::post('waitingrooms/{waitingrooms}/split', 'WaitingRoomsController@split')->name('split');
```

Comment:
It might have? I changed that (to what you suggested) while I was debugging something else. I didn't feel like it was a problem since it did the same thing, but ¯\_(ツ)_/¯